examples/pytorch/graphsage/experimental/README.md

```diff
@@ -28,15 +28,31 @@ will be able to access the partitioned data.
 
 We need to run a server on each machine. Before running the servers, we need to update `ip_config.txt` with the right IP addresses.
 
+On each of the machines, set the following environment variables.
+
+```bash
+export DGL_ROLE=server
+export DGL_IP_CONFIG=ip_config.txt
+export DGL_CONF_PATH=data/ogb-product.json
+export DGL_NUM_CLIENT=4
+```
+
 ```bash
 # run server on machine 0
-python3 train_dist.py --server --graph-name ogb-product --id 0 --num-client 4 --conf_path data/ogb-product.json --ip_config ip_config.txt
+export DGL_SERVER_ID=0
+python3 train_dist.py
```

Comment:
Can we use `python3 DGL_SERVER_ID=0 train_dist.py` here? In such way we can launch many server processes in the same machine.

Reply:
sure. this is temporary modification. once, you finish launch script, we don't need users to run any of the commands.